src/main/java/com/celements/javascript/JsLoadMode.java

```diff
@@ -0,0 +1,21 @@
+package com.celements.javascript;
+
+import javax.annotation.Nullable;
+import javax.validation.constraints.NotNull;
+
+import com.google.common.base.Strings;
+
+public enum JsLoadMode {
+
+  SYNC, DEFER, ASYNC;
+
+  @NotNull
+  public static JsLoadMode convertStoreValue(@Nullable String storeValue) {
+    try {
+      return valueOf(Strings.nullToEmpty(storeValue).toUpperCase());
+    } catch (IllegalArgumentException exp) {
+      return SYNC;
+    }

```

Comment:
with guava `com.google.common.base.Enums`
```suggestion
    return Enums.getIfPresent(Strings.nullToEmpty(storeValue).toUpperCase())
        .or(SYNC);
```